src/baseline_bots/utils.py

```diff
@@ -117,22 +131,41 @@ def parse_PRP(msg) -> str:
         raise Exception(f"Cant parse PRP msg {msg}")
 
 
-def parse_orr_xdo(msg: str) -> List[str]:
+def parse_arrangement(msg: str, xdo_only=True) -> List[str]:
     """
-    Attempts to parse a specific message configuration
+    Attempts to parse arrangements (may or may not have ORR keyword)
+
+    Examples when xdo_only = False
+    XDO (F BLA - CON) -> ("XDO", "F BLA - CON")
+    ORR (XDO ((RUS FLT BLA) MTO CON)) (ALY (GER RUS TUR) VSS (FRA ENG ITA AUS)) (ABC (F BLA - CON))
```

Comment:
Do we expect to receive XDO and ALY at once? I don't think this is possible in DAIDE syntax

Reply:
To be honest, I am not sure if such messages are common. But the syntax is valid and possible. Check https://github.com/SHADE-AI/daidepp/blob/main/src/daidepp/grammar.py

They say:
```
"orr": '"ORR" lpar sub_arrangement rpar (lpar sub_arrangement rpar)+',
"sub_arrangement": "pce / aly_vss / drw / slo / not / nar / mto / xdo / dmz",
```

This means all such sub arrangements can be included in an ORR statement.

I just tried making our code future proof. Just in case, any power sends out such messages, we are ready to process it.